cryptor/cryptor.go

```diff
@@ -742,6 +742,11 @@ var ErrRestoreDelegations = errors.New("cryptor: need more delegations")
 // enough delegations are present to restore the cache, the current
 // Red October key cache is replaced with the persisted one.
 func (c *Cryptor) Restore(name, password string, uses int, slot, durationString string) error {
+	// If the persistence store is already active, don't proceed.
+	if st := c.persist.Status(); st != nil && st.State == persist.Active {
+		return nil
```

Comment:
Should this be returned as a designated error and caught? It seems different from the other success case, but perhaps the caller is never interested in this fact?